lib/apiservers/portlayer/swagger.yml

```diff
@@ -859,3 +920,46 @@ definitions:
       state:
         type: string
         enum: ["RUNNING", "STOPPED"]
+  VolumeRequest:
+    type: object
+    required:
+      - Name
+      - Store
+      - Driver
+    properties:
+      Name:
+        type: string
+      Driver:
```

Comment:
the portlayer does not care about the driver name. Remove this from the inputs. Since the portlayer is our driver. Per conversation with george (6/22)